native/src/main/java/io/ballerina/lib/ldap/Client.java

```diff
@@ -171,14 +183,14 @@ public static Object compare(Environment env, BObject ldapClient,
         return null;
     }
 
-    public static Object getEntry(BObject ldapClient, BString distinguishedName, BTypedesc typeParam) {
+    public static Object getEntry(BObject ldapClient, BString dN, BTypedesc typeParam) {
         BMap<BString, Object> entry = ValueCreator.createMapValue();
         try {
             LDAPConnection ldapConnection = (LDAPConnection) ldapClient.getNativeData(NATIVE_CLIENT);
             validateConnection(ldapConnection);
-            SearchResultEntry userEntry = ldapConnection.getEntry(distinguishedName.getValue());
+            SearchResultEntry userEntry = ldapConnection.getEntry(dN.getValue());
             if (userEntry == null) {
-                return Utils.createError(ENTRY_NOT_FOUND + distinguishedName, new LDAPException(NO_SUCH_OBJECT));
+                return Utils.createError(ENTRY_NOT_FOUND + dN + "'", new LDAPException(NO_SUCH_OBJECT));
```

Comment:
Shouldn't this be
```suggestion
                return Utils.createError(ENTRY_NOT_FOUND + + "'" + dN + "'", new LDAPException(NO_SUCH_OBJECT));
```
?

Reply:
The value for the ENTRY_NOT_FOUND constant is provided as "Entry is not found for DN: '" (with the colon). Is that okay?

Reply:
Yeah thats also fine, although having such "unbalanced" common constants might look confusing in its usages IMO.

Reply:
Fixed in [0474d7f](https://github.com/ballerina-platform/module-ballerina-ldap/pull/51/commits/0474d7fe46f4d3d822fa11fb29759107933bb6ca)